Tasks/AzureWebAppContainerV1/taskparameters.ts

```diff
@@ -36,6 +39,18 @@ export class TaskParametersUtility {
         var endpointTelemetry = '{"endpointId":"' + taskParameters.connectedServiceName + '"}';
         console.log("##vso[telemetry.publish area=TaskEndpointId;feature=AzureRmWebAppDeployment]" + endpointTelemetry);
 
+        taskParameters.isMultiContainer = false;
+        taskParameters.isMultiContainer = taskParameters.ImageName && taskParameters.ImageName.indexOf("\n") !=-1;
```

Comment:
if there is a valid file provided in config file, then also it is multicontainer app

Reply:
It is done.

Reply:
This is something we should discuss with PMs. Because then imageName input can be optional.

Reply:
It should be optional. In KubernetesManifest task as well, the containers input is optional. One needs to provide a value for this input only if they want to perform tag/sha256 substitutions for the images specified in the docker compose file.

Reply:
We had in mandatory in our task since for single container scenarios it is required. Making it optional will allow the user to execute the task with invalid inputs though we will catch it at run time.